src/main/resources/META-INF/archetype-post-generate.groovy

```diff
@@ -30,6 +32,13 @@ if (optionAemVersion == "6.3.3") {
     assert new File(uiContentPackage, "src/main/content/jcr_root/conf/" + confFolderName  + "/settings/wcm/segments").deleteDir()
 }
 
+def sdkVersion
+if (optionAemVersion == "cloud") {
+    sdkVersion = getLatestSDK("Test")
```

Comment:
Why is the parameter "Test" passed to the `getLatestSKD` method?

Reply:
Good catch. The intention was to pass the version of the archetype when querying for the latest SDK version, so we have an idea about usage from developers. Forgot to update this.

Reply:
Fixed in https://github.com/adobe/aem-project-archetype/pull/334/commits/dd99994275794081fba43dc4c39bf3048aacff36